hack/check-pr.sh

```diff
@@ -21,24 +21,8 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-dirs=()
-tests=()
-ref="${1:-HEAD}"
-echo -n "Packages changed since $ref: "
-for d in $(git diff --name-only "$ref" | xargs -n 1 dirname | sort -u); do
-    if ! ls "./$d/"*.go &> /dev/null; then
-        continue
-    fi
-    echo -n "$d "
-    dirs+=("./$d")
-    tests+=("//$d:all")
-done
-
-if [[ ${#dirs[@]} == 0 ]]; then
-    echo NONE
-    exit 0
-fi
-echo
+cd $(git rev-parse --show-toplevel)
```

Comment:
How about we start with just this change?

I don't think the rest justifies rewriting this script.

Reply:
ok
/shrug